requirements.txt

```diff
@@ -1,7 +1,7 @@
 astropy
 pytz
-scipy
-healpy
+scipy<1.14.0
```

Comment:
is this strongly required? Because we might encounter problems between libraries in the future...